src/app/zap-templates/zcl/data-model/silabs/ha.xml

```diff
@@ -1491,25 +1491,25 @@ limitations under the License.
       <description>
         Indicates lock success or failure
       </description>
-      <arg name="status" type="INT8U"/>
+      <arg name="status" type="Status"/>
```

Comment:
Are these changes needed for this PR?  That is, are these cleanup, or are they necessary for some reason to make things compile?  If the latter, why?